model_analyzer/cli/cli.py

```diff
@@ -116,16 +116,19 @@ def _add_arguments(self):
         )
         self._parser.add_argument(
             '--triton-version',
+            default='20.11-py3',
```

Comment:
This is fine for now, but I would prefer not to have to maintain this for every release.